tests/unit/bokeh/test_resources.py

```diff
@@ -455,6 +456,32 @@ class AClass(Model):
     assert r.css_files.index("external_css_3") > r.css_files.index("external_css_2")
     assert r.js_files.index("external_js_3") > r.js_files.index("external_js_2")
 
+
+def test_Resources_with_BOKEH_MINIFIED() -> None:
+    with envset(BOKEH_MINIFIED="yes"):
+        r = resources.Resources()
+        assert r.minified is True
+
+    with envset(BOKEH_MINIFIED="no"):
+        r = resources.Resources()
+        assert r.minified is False
+
+    with envset(BOKEH_MINIFIED="yes", BOKEH_DEV="yes"):
+        r = resources.Resources()
+        assert r.minified is True
+
+    with envset(BOKEH_MINIFIED="yes", BOKEH_DEV="no"):
+        r = resources.Resources()
+        assert r.minified is True
+
+    with envset(BOKEH_MINIFIED="no", BOKEH_DEV="yes"):
+        r = resources.Resources()
+        assert r.minified is True
+
+    with envset(BOKEH_MINIFIED="no", BOKEH_DEV="no"):
+        r = resources.Resources()
+        assert r.minified is False
+
```

Comment:
Since default mode is cdn, suggest adding explicit test(s) with mode set to inline (since that was the actual reported issue case)

Reply:
I parameterized this test on `mode`.